apps/web/pages/api/book/event.ts

```diff
@@ -443,70 +428,96 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     });
 
     const credentials = currentUser.credentials;
+    const calendarBusyTimes: EventBusyDate[] = await prisma.booking
```

Comment:
Not a blocking issue but we should strive to rely on inferred types as much as possible.